src/twisted/trial/_dist/workercommands.py

```diff
@@ -15,14 +15,14 @@ class Run(Command):
     """
     Run a test.
     """
-    arguments = [('testCase', String())]
-    response = [('success', Boolean())]
+    arguments = [(b'testCase', String())]
```

Comment:
If we switched this to use `t.p.amp.Unicode` instead, would we be able to avoid the Unicode Sandwich in `t.t._dist.workerreporter` while retaining compatibility with Python 2.x?

Reply:
It should be only Unicode on Python 3 since `TestCase`, `Failure`, etc. define these in terms of native strings.